src/component/vocabulary/CreateVocabularyForm.tsx

```diff
@@ -194,6 +265,28 @@ const CreateVocabularyForm: React.FC<CreateVocabularyFormProps> = ({
                   />
                 </Col>
               </Row>
+              <Row>
```

Comment:
Let's move this just below the vocabulary description textarea (applies to vocabulary edit as well).